server/mcp_server_askecho/pyproject.toml

```diff
@@ -0,0 +1,20 @@
+[project]
+name = "mcp-server-askecho"
+version = "0.1.0"
+description = "MCP Server for AskEcho"
+readme = "README.md"
+authors = [
+    { name = "liqi.0117", email = "liqi.0117@bytedance.com" }
+]
+requires-python = ">=3.13"
+dependencies = [
+    "mcp>=1.9.3",
```

Comment:
Using this version of mcp  means no oauth support any more in the future. askecho could kept at local and cloud deployment only.